[2024-10-02T17:25:05.774+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T17:25:06.149+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.extrair scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2024-10-02T17:25:06.158+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.extrair scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2024-10-02T17:25:06.159+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T17:25:06.185+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extrair> on 2023-10-06 00:00:00+00:00
[2024-10-02T17:25:06.196+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1176) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T17:25:06.199+0000] {standard_task_runner.py:63} INFO - Started process 1205 to run task
[2024-10-02T17:25:06.201+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'extrair', 'scheduled__2023-10-06T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpue6ns833']
[2024-10-02T17:25:06.205+0000] {standard_task_runner.py:91} INFO - Job 10: Subtask extrair
[2024-10-02T17:25:06.290+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.extrair scheduled__2023-10-06T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T17:25:06.402+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='extrair' AIRFLOW_CTX_EXECUTION_DATE='2023-10-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-06T00:00:00+00:00'
[2024-10-02T17:25:06.406+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T17:25:13.676+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T17:25:13.676+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T17:25:13.685+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=extrair, run_id=scheduled__2023-10-06T00:00:00+00:00, execution_date=20231006T000000, start_date=20241002T172506, end_date=20241002T172513
[2024-10-02T17:25:13.714+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T17:25:13.736+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-02T17:25:13.740+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-10-02T18:57:07.217+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T18:57:07.245+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.extrair scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2024-10-02T18:57:07.253+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.extrair scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2024-10-02T18:57:07.254+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T18:57:07.542+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extrair> on 2023-10-06 00:00:00+00:00
[2024-10-02T18:57:07.548+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=5258) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T18:57:07.549+0000] {standard_task_runner.py:63} INFO - Started process 5275 to run task
[2024-10-02T18:57:07.550+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'extrair', 'scheduled__2023-10-06T00:00:00+00:00', '--job-id', '973', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp8hrh3qgi']
[2024-10-02T18:57:07.552+0000] {standard_task_runner.py:91} INFO - Job 973: Subtask extrair
[2024-10-02T18:57:07.600+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.extrair scheduled__2023-10-06T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T18:57:07.688+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='extrair' AIRFLOW_CTX_EXECUTION_DATE='2023-10-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-06T00:00:00+00:00'
[2024-10-02T18:57:07.690+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T18:57:13.009+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T18:57:13.010+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T18:57:13.017+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=extrair, run_id=scheduled__2023-10-06T00:00:00+00:00, execution_date=20231006T000000, start_date=20241002T185707, end_date=20241002T185713
[2024-10-02T18:57:13.048+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T18:57:13.066+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-02T18:57:13.068+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-10-02T19:23:11.708+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T19:23:11.734+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.extrair scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2024-10-02T19:23:11.743+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.extrair scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2024-10-02T19:23:11.743+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T19:23:12.006+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extrair> on 2023-10-06 00:00:00+00:00
[2024-10-02T19:23:12.013+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=8715) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T19:23:12.015+0000] {standard_task_runner.py:63} INFO - Started process 8732 to run task
[2024-10-02T19:23:12.014+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'extrair', 'scheduled__2023-10-06T00:00:00+00:00', '--job-id', '2013', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpvvspwwdo']
[2024-10-02T19:23:12.020+0000] {standard_task_runner.py:91} INFO - Job 2013: Subtask extrair
[2024-10-02T19:23:12.075+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.extrair scheduled__2023-10-06T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T19:23:12.164+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='extrair' AIRFLOW_CTX_EXECUTION_DATE='2023-10-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-06T00:00:00+00:00'
[2024-10-02T19:23:12.165+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T19:23:23.176+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T19:23:23.177+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/etl_pipeline.py", line 9, in extrair
    df = pd.read_csv(url)
         ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/io/parsers/readers.py", line 948, in read_csv
    return _read(filepath_or_buffer, kwds)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/io/parsers/readers.py", line 611, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/io/parsers/readers.py", line 1448, in __init__
    self._engine = self._make_engine(f, self.engine)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/io/parsers/readers.py", line 1705, in _make_engine
    self.handles = get_handle(
                   ^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/io/common.py", line 718, in get_handle
    ioargs = _get_filepath_or_buffer(
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/io/common.py", line 377, in _get_filepath_or_buffer
    reader = BytesIO(req.read())
                     ^^^^^^^^^^
  File "/usr/local/lib/python3.12/http/client.py", line 495, in read
    s = self._safe_read(self.length)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/http/client.py", line 642, in _safe_read
    raise IncompleteRead(data, amt-len(data))
http.client.IncompleteRead: IncompleteRead(2262039 bytes read, 2045263 more expected)
[2024-10-02T19:23:23.190+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=etl_covid_europa_pipeline, task_id=extrair, run_id=scheduled__2023-10-06T00:00:00+00:00, execution_date=20231006T000000, start_date=20241002T192311, end_date=20241002T192323
[2024-10-02T19:23:23.202+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 2013 for task extrair (IncompleteRead(2262039 bytes read, 2045263 more expected); 8732)
[2024-10-02T19:23:23.239+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-10-02T19:23:23.258+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T19:23:23.263+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
